packages/api-plugin-fulfillment-method-pickup-store/src/index.js

```diff
@@ -0,0 +1,36 @@
+import { createRequire } from "module";
+import fulfillmentMethodPickupStorePreStartup from "./preStartup.js";
+import fulfillmentMethodPickupStoreStartup from "./startup.js";
+import { MethodStoreData } from "./simpleSchemas.js";
+import schemas from "./schemas/index.js";
+import getFulfillmentMethodsWithQuotesPickupStore from "./getFulfillmentMethodsWithQuotesPickupStore.js";
+import validateOrderMethodsstore from "./util/validateOrderMethodsstore.js";
+
+const require = createRequire(import.meta.url);
+const pkg = require("../package.json");
```

Comment:
Are we missing an export of `logCtx` here?

Reply:
As in other plugins, I have declared the `logCtx` only in those files where it is used

Reply:
I think you misunderstood. Why are you declaring it here if you don't plan to use it elsewhere since you don't use it here? Otherwise you should remove it.

Reply:
@zenweasel , are you referring to the `const pkg = require("../package.json");`
That is used to get the `pkg.version` used in the registerPlugin